plugins/usepe_logger.py

```diff
@@ -63,3 +74,34 @@ def update(self):
 
         # Store the new conflict environment
         self.prevconf = currentconf
+    
+    def los_logger(self):
+        currentlos = list()
+
+        # Go through all loss of separation pairs and sort the IDs for easier matching
+        currentlos = [list(pair) for pair in traf.cd.lospairs_unique]
+        for pair in currentlos: pair.sort()
```

Comment:
Replacing `list()` with `sorted()` would get rid of the `for` loop I think:
```suggestion
        currentlos = [sorted(pair) for pair in traf.cd.lospairs_unique]
```

Reply:
That works!